src/assets/toolkit/scripts/home-animation.js

```diff
@@ -0,0 +1,300 @@
+/**
+ * GSAP is powerful and performant, but its organization is a little old-school.
+ * This imports a bunch of functions to the global scope.
+ *
+ * See: https://github.com/greensock/GreenSock-JS/issues/71
+ */
+
+import TweenLite from 'gsap/src/uncompressed/TweenLite';
+import TimelineMax from 'gsap/src/uncompressed/TimelineMax';
+import EasePack from 'gsap/src/uncompressed/easing/EasePack';
+import CSSPlugin from 'gsap/src/uncompressed/plugins/CSSPlugin';
+import AttrPlugin from 'gsap/src/uncompressed/plugins/AttrPlugin';
+
+/**
+ * Settings
+ */
+
+const settings = {
+  morph: {
+    delay: '+=1',
+    duration: 0.5,
+    ease: Elastic.easeOut.config(0.5, 0.4)
+  }
+};
+
+/**
+ * Timelines
+ */
+
+const timelines = {
+  morph: new TimelineMax({
+    repeat: -1
+  })
+};
+
+/**
+ * Elements
+ */
+
+const dom = (function (nodeList) {
+  const result = {};
+  nodeList = Array.prototype.slice.call(nodeList);
```

Comment:
Yes, but not in this context. The SVG is its own document.

That said, I'm glad you mentioned this because the line isn't even necessary... I'm just using a `for` loop, so I don't need to convert the `nodeList` at all.